app/views/broadcast_announcements/_broadcast_announcement.html.erb

```diff
@@ -0,0 +1,14 @@
+<tr id="<%= dom_id broadcast_announcement %>">
+  <td><%= broadcast_announcement.message %></td>
+  <td><%= link_to  broadcast_announcement.link, broadcast_announcement.link %></td>
+  <td><%= User.find(broadcast_announcement.user_id).name %></td>
+  <td><%= broadcast_announcement.expiry %>
+    <% unless broadcast_announcement.expiry.nil? || broadcast_announcement.expiry.strftime('%y %m %d') >= DateTime.now.strftime('%y %m %d') %>
+      <span class="badge badge-danger">Expired</span>
+    <% end %>
```

Comment:
This should be a method called \`expired?\` on the `BroadcastAnnouncement` model.